__tests__/server/middleware/createRequestHtmlFragment.spec.js

```diff
@@ -14,24 +14,35 @@
  * permissions and limitations under the License.
  */
 
+/* global BigInt */
+
 import url from 'url';
 import { browserHistory } from '@americanexpress/one-app-router';
 import { Map as iMap, fromJS } from 'immutable';
+import { composeModules } from 'holocron';
 import match from '../../../src/universal/utils/matchPromisified';
+import breaker from '../../../src/server/utils/circuitBreaker';
 
 import * as reactRendering from '../../../src/server/utils/reactRendering';
 
 jest.mock('../../../src/universal/utils/matchPromisified');
 
 jest.mock('holocron', () => ({
   composeModules: jest.fn(() => 'composeModules'),
+  getModule: () => () => 0,
 }));
 
+const sleep = (ms) => new Promise((res) => setTimeout(res, ms));
 const renderForStringSpy = jest.spyOn(reactRendering, 'renderForString');
 const renderForStaticMarkupSpy = jest.spyOn(reactRendering, 'renderForStaticMarkup');
+const msToNs = (n) => n * 1e6;
 
 describe('createRequestHtmlFragment', () => {
   jest.spyOn(console, 'error').mockImplementation(() => {});
+  const realHrtime = process.hrtime;
+  const mockHrtime = (...args) => realHrtime(...args);
+  mockHrtime.bigint = jest.fn();
+  process.hrtime = mockHrtime;
```

Comment:
I think this should not really required here, e.g. the breaker should be automatically disabled in tests.

Reply:
This is just needed for the tests that are asserting on the functionality of the breaker